code/modules/mob/living/simple_animal/abnormality/waw/warden.dm

```diff
@@ -0,0 +1,97 @@
+/mob/living/simple_animal/hostile/abnormality/warden
+	name = "The Warden"
+	desc = "An abnormality that takes the form of a fleshy stick wearing a dress and eyes. You don't want to know what's under that dress."
+	icon = 'ModularTegustation/Teguicons/48x64.dmi'
+	icon_state = "warden"
+	icon_living = "warden"
+	maxHealth = 1700
+	health = 1700
+	pixel_x = -8
+	base_pixel_x = -8
+	damage_coeff = list(BRUTE = 0.7, RED_DAMAGE = 0.7, WHITE_DAMAGE = 1.2, BLACK_DAMAGE = 0.4, PALE_DAMAGE = 1.2)
+
+	melee_damage_lower = 70
+	melee_damage_upper = 70
+	melee_damage_type = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	stat_attack = HARD_CRIT
+	attack_sound = 'sound/weapons/slashmiss.ogg'
+	attack_verb_continuous = "claws"
+	attack_verb_simple = "claws"
+	faction = list("warden")	//It should attack everything else, fuck you.
+	can_breach = TRUE
+	threat_level = WAW_LEVEL
+	start_qliphoth = 2
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = 40,
+						ABNORMALITY_WORK_INSIGHT = 15,
+						ABNORMALITY_WORK_ATTACHMENT = 0,
+						ABNORMALITY_WORK_REPRESSION = 50
+						)
+	work_damage_amount = 7
+	work_damage_type = BLACK_DAMAGE
+
+	attack_action_types = list()
+
+	ego_list = list(
+		)
+
+	var/finishing = FALSE
+
+/mob/living/simple_animal/hostile/abnormality/warden/AttackingTarget()
+	. = ..()
+	if(.)
+		if(!istype(target, /mob/living/carbon/human))
+			return
+		var/mob/living/carbon/human/H = target
+
+		if(H.health < 0)
+
+			finishing = TRUE
+			icon_state = "warden_attack"
+			playsound(get_turf(src), 'sound/hallucinations/wail.ogg', 50, 1)
+			SLEEP_CHECK_DEATH(5)
```

Comment:
Since you are placing a sleep() here, add a check for H being deleted or moved away, just like with scarecrow.

Reply:
Honestly, its so fast that the dusting animation cant even finish before she eats you. It's a non-issue imo.
The sleep check is mostly for the animation and to stop her while she does it